tests/phpunit/elementor/modules/atomic-widgets/styles/styles-transformers/test-linked-dimensions-transformer.php

```diff
@@ -23,9 +22,7 @@ public function test_transform__returns_linked_dimensions() {
 			'right' => '20px',
 			'bottom' => '30px',
 			'left' => '40px',
-		], function( $value ) {
```

Comment:
I think we should delete all the test for specific transformer and try to test the main flow, if you wish lets talk about it